### ISSUE TITLE
Provide the ability to lazily yield rows from Table storage

### ISSUE THREAD

**Issue by isaacabraham** [fsprojects/AzureStorageTypeProvider#40]
This can be accomplished using the continuation token in Azure Table Storage.

**Comment by isaacabraham** [comment]
Not sure how this would work in the context of Async though.

**Comment by stewart-r** [comment]
Something like this?

http://tomasp.net/blog/async-sequences.aspx/

**Comment by isaacabraham** [comment]
Yeah, I had thought of that. I think it's probably overkill to add an entire CE just for this though. An alternative is to simply leaves things as they are or to only add it in for the synchronous version.
